kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -397,7 +397,7 @@ public T createOrReplace(T... items) {
       throw new IllegalArgumentException("Nothing to create.");
     }
 
-    if (Utils.isNullOrEmpty(name) && item instanceof HasMetadata) {
+    if (Utils.StringUtils.isNullOrEmpty(name) && item instanceof HasMetadata) {
```

Comment:
If you use the full name, you don't need to import statically. Same for the other cases in the source code.
It should be enough to do `impor io.fabric8.kubernetes.client.utils.Utils.StringUtils;`